op-e2e/system_fpp_test.go

```diff
@@ -93,7 +93,7 @@ func testVerifyL2OutputRootEmptyBlock(t *testing.T, detached bool) {
 	l2OutputRoot := agreedL2Output.OutputRoot
 
 	t.Log("=====Stopping batch submitter=====")
-	err = sys.BatchSubmitter.Stop(ctx)
+	err = sys.BatchSubmitter.Driver().StopBatchSubmitting(context.Background())
```

Comment:
There's a `ctx` available here, so I'd use it. Although it is just the background context right now, if it is ever changed, we don't miss using it here.